spec.bs

```diff
@@ -1819,9 +1832,14 @@ To <dfn>report win</dfn> given a [=leading bid info=] |leadingBidInfo|, a [=stri
   1. If the result of [=query reporting ID k-anonymity count=] given |winner|'s [=generated bid/interest group=]
 
     and |winner|'s [=generated bid/ad descriptor=]'s [=ad descriptor/url=] is true:
-
-    1. [=map/Set=] |browserSignals|["`interestGroupName`"] to |winner|'s
-      [=generated bid/interest group=] [=interest group/name=].
+    1. Let |igAd| be the [=interest group ad=] from |winner|'s [=generated bid/interest group=]'s [=interest group/ads=] whose [=interest group ad/render url=] is |winner|'s [=generated bid/ad descriptor=]'s [=ad descriptor/url=].
+    1. If |igAd|'s [=interest group ad/buyer and seller reporting ID=]
+          [=map/exists=]:
+      1. [=map/Set=] |browserSignals|["`buyerAndSellerReportingId`"] to |igAd|'s [=interest group ad/buyer and seller reporting ID=].
+    1. Otherwise, if |igAd|'s [=interest group ad/buyer reporting ID=]
+          [=map/exists=]
+      1. [=map/Set=] |browserSignals|["`buyerReportingId`"] to |igAd|'s [=interest group ad/buyer reporting ID=].
+    1. Otherwise, [=map/Set=] |browserSignals|["`interestGroupName`"] to |winner|'s [=generated bid/interest group=] [=interest group/name=].

```

Comment:
I just merged a [PR](https://github.com/WICG/turtledove/pull/678/files#diff-6f5a1d8263b0b0c42e2716ba5750e3652e359532647ac934c1c70086ae3ceddaR1855) that changes |browserSignals| to webIDL, so some of the field setting steps need to be updated. Sorry about that. And remove this [todo](https://github.com/WICG/turtledove/pull/678/files#diff-6f5a1d8263b0b0c42e2716ba5750e3652e359532647ac934c1c70086ae3ceddaR1855) I added in that PR.

Reply:
Should I hold off reviewing until the merge conflict is fixed @morlovich?

Reply:
Yes, the changes required are substantative.